acceptance/framework/helpers/helpers.go

```diff
@@ -358,3 +360,48 @@ func createCmdArgs(options *k8s.KubectlOptions) []string {
 	}
 	return cmdArgs
 }
+
+const DEFAULT_PAUSE_PORT = ":8517"
```

Comment:
~ Should we consider a higher port range for this than `8500`s, so that we're out of the "immediately likely next ports" that Consul/related tools might use in normal prod operations? e.g. `38501`